inc/cachify.class.php

```diff
@@ -639,6 +639,12 @@ public static function add_dashboard_count( $items = array() )
 			)
 		);
 
+		/* Output of the cache size */
+		$cachesize = ( $size == 0 )
+			? esc_html__( 'Empty Cache', 'cachify' ) :
+			/* translators: %s: cache size */
+			sprintf( esc_html__( '%s Cache', 'cachify' ), size_format($size) );;
```

Comment:
think it is nicer to have $cachesize in front
but if all of you want we can do it this way and i will upload this:

'''
if ( empty( $size ) ) {
			$cachesize = esc_html__( 'Empty Cache', 'cachify' );
			}
			else {
				$cachesize =
				/* translators: %s: cache size */
				sprintf( esc_html__( '%s Cache', 'cachify' ), size_format($size) );
				}
'''